modules/microshift-storage-about-vol-snapshots.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// microshift/creating-volume-snapshots-microshift.adoc
+
+:_content-type: CONCEPT
+[id="about-volume-snapshots_{context}"]
+= About volume snapshots
+
+Volume snapshots can be used with LVM thin volumes to help protect against data loss in {product-title}. Only the `microshift csi` container storage interface (CSI) is supported.
```

Comment:
May be this line "Only the `microshift csi` container storage interface (CSI) is supported.", can be rephrased to "Only supported on Microshift default  LVMS container storage interface (CSI)"